src/main/haskell/language-kore/test/parser/Data/Kore/Unparser/UnparseTest.hs

```diff
@@ -113,14 +114,15 @@ unparseUnitTests =
             ++ "    module i\n    endmodule\n    []\n"
             ++ "    module k\n    endmodule\n    []\n"
             )
-        , unparseTest
-            ( asSentence SentenceImport
-                { sentenceImportModuleName = ModuleName {getModuleName = "sl"}
-                , sentenceImportAttributes =
-                    Attributes { getAttributes = [] } :: KoreAttributes
-                }
-            )
-            "import sl[]"
+            -- FIXME: Put this test back in. What Unparse instance does it want?
+        -- , unparseTest
+        --     ( constructUnifiedSentence SentenceImportSentence $ SentenceImport
+        --         { sentenceImportModuleName = ModuleName {getModuleName = "sl"}
+        --         , sentenceImportAttributes =
+        --             Attributes { getAttributes = [] } :: Attributes
+        --         }
+        --     )
+        --     "import sl[]"
```

Comment:
should work for unparse of `KoreSentence`, shouldn't it?

Reply:
Not sure what to do about this test. The error i get is "Ambiguous type variable `sortParam0' arising from a use of `unparseTest' prevents the constraint (Unparse sortParam0) from being solved."

Overall, the problem is that Attributes no longer carries the type variables `pat' and `var', resulting in fewer constraints when inferring typeclass instances in a function which takes something with Attributes. However, this is sortParam not pat or var, so I'm not why this happens.

Reply:
Have you tried typing the entire sentence to something like `KoreSentence` ?  This should default sortParam to something...

Reply:
Yup, I think that worked